content/master/concepts/composite-resources.md

```diff
@@ -0,0 +1,470 @@
+### Composition revision policy
+
+If Crossplane uses [Composition revisions]({{<ref "/knowledge-base/guides/composition-revisions">}}) a composite resource can use
+a {{<hover label="comprev" line="6">}}compositionUpdatePolicy{{</hover>}} to
+manually or automatically reference newer Composition revisions.
+
+{{<hint "important" >}}
+Composition revisions are an alpha feature. Alpha features aren't enabled by
+default. 
+{{< /hint >}}
+
+The default 
+{{<hover label="comprev" line="6">}}compositionUpdatePolicy{{</hover>}} is 
+"Automatic." Composite resources automatically use the latest Composition
+revision. 
+
+Change the policy to 
+{{<hover label="comprev" line="6">}}Manual{{</hover>}} to prevent composite
+resources from automatically upgrading.
+
+```yaml {label="comprev",copy-lines="none"}
+apiVersion: example.org/v1alpha1
+kind: xMyDatabase
+metadata:
+  name: my-composite-resource
+spec:
+  compositionUpdatePolicy: Manual
+  # Removed for brevity
+```
+
+### Composition revision selection
+If Crossplane uses [Composition revisions]({{<ref "/knowledge-base/guides/composition-revisions">}}) a composite resource can
+select a specific Composition revision.
+
+{{<hint "important" >}}
+Composition revisions are an alpha feature. Alpha features aren't enabled by
+default. 
+{{< /hint >}}
+
+Use {{<hover label="comprevref" line="6">}}compositionRevisionRef{{</hover>}} to
+select a specific Composition revision by name.
+
+For example, to select a specific Composition revision use the name of the
+desired Composition revision. 
+
+```yaml {label="comprevref",copy-lines="none"}
+apiVersion: example.org/v1alpha1
+kind: xMyDatabase
+metadata:
+  name: my-composite-resource
+spec:
+  compositionRevisionRef: my-composition-b5aa1eb
+  # Removed for brevity
+```
+
+{{<hint "note" >}}
+Find the Composition revision name from 
+{{<hover label="getcomprev" line="1">}}kubectl get compositionrevision{{</hover>}}
+
+```shell {label="getcomprev",copy-lines="1"}
+kubectl get compositionrevision
+NAME                         REVISION   XR-KIND        XR-APIVERSION            AGE
+my-composition-5c976ad       1          xmydatabases   example.org/v1alpha1     65m
+my-composition-b5aa1eb       2          xmydatabases   example.org/v1alpha1     64m
+```
+{{< /hint >}}
+
+A Composite resource can also select Composition revisions based on labels
+instead of the exact name with a 
+{{<hover label="comprevsel" line="6">}}compositionRevisionSelector{{</hover>}}.
+
+Inside the {{<hover label="comprevsel" line="7">}}matchLabels{{</hover>}} 
+section provide one or more Composition revision labels to match.
+
+
+```yaml {label="comprevsel",copy-lines="none"}
+apiVersion: example.org/v1alpha1
+kind: xMyDatabase
+metadata:
+  name: my-composite-resource
+spec:
+  compositionRevisionSelector:
+    matchLabels:
+      channel: dev
+  # Removed for brevity
+```
+
+### Manage connection secrets 
+
+When a composite resource creates managed resources, Crossplane provides any
```

Comment:
i was using the terminology "child composed resource" in https://github.com/crossplane/docs/pull/485 and https://blog.crossplane.io/faq-2-claim-connection-details/, because the composition could be creating MRs but also nested XRs as well.  Do you want to make that same distinction or use similar terminology?

Reply:
I tried to avoid calling out "managed resources" for that reason but since we're talking about connection secrets only the MR can generate them, right? 

If I'm nesting XRs can the XR generate secrets?

Reply:
yes the nested XRs can generate/propagate their own connection details up the chain too!

here's a tangible example in platform-ref-aws where the XEKS composed resource is delcaring/propagating connection details:
https://github.com/upbound/platform-ref-aws/blob/main/package/cluster/composition.yaml#L28-L32